packages/boot/test/bootstrapTests/supports.ts

```diff
@@ -215,13 +215,23 @@ export const makeProposalExtractor = ({ childProcess, fs }: Powers) => {
   const getPkgPath = (pkg, fileName = '') =>
     new URL(`../../../${pkg}/${fileName}`, import.meta.url).pathname;
 
-  const runPackageScript = async (pkg, name, env) => {
-    console.warn(pkg, 'running package script:', name);
-    const pkgPath = getPkgPath(pkg);
-    return childProcess.execFileSync('yarn', ['run', name], {
-      cwd: pkgPath,
+  const importSpec = spec =>
+    importMetaResolve(spec, import.meta.url).then(u => new URL(u).pathname);
+
+  const runPackageScript = async (outputDir, scriptPath, env) => {
+    console.warn('running package script:', scriptPath);
+    const out = await childProcess.execFileSync('yarn', ['bin', 'agoric'], {
```

Comment:
not a blocker but we shouldn't need to shell out here. `yarn bin agoric` is running https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/src/entrypoint.js#L53 which is just a wrapper around the JS module https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/src/main.js that could be called here as a function